templates/bridge-error.html.php

```diff
@@ -0,0 +1,43 @@
+<section>
```

Comment:
We might want to group templates somehow, eg. move them into `error` subdirectory.

Reply:
When we have more than 20 templates we can start making sub-directories. Not before then.